frontend/angular.json

```diff
@@ -30,7 +30,6 @@
               "chartjs-plugin-datalabels"
             ],
             "preserveSymlinks": true,
-            "aot": true,
```

Comment:
I was wondering why this got removed, but it's the following migration:

```
▸ Remove deprecated options from 'angular.json' that are no longer present in v12.
  UPDATE angular.json (4694 bytes)
  Migration completed.
``` 

The options are not present in the schema and are unused anyway